README.md

```diff
@@ -16,7 +16,48 @@ Further documentation will be available on
 # Please note
 This version works only with `python 3.6.x`
 
+- [Get Started](#get-started)
+- [How to contribute](#how-to-contribute)
+- [Maintenance](#Maintenance)
+- [License](#license)
 
+# Get Started
 
+This is the main codebase for FARE. FARE is an e-learning web platform. Inside FARE you can found a lot of educational material, download it or you can upload some.
 
+## Searching material
+To search the desired material you just have to type what you are looking for in the search bar
+
+## Upload content
+Everyone can upload content, you have to be logged in. After the upload your content will be reviewed and, if everything is ok, it will be published
+
+## Review the material
+If you are a staff member, you can review the material and let it be visible by everyone!
+
+# How to contribute
+
+This project runs Invenio vx.x (insert structure of the project).
+To make run our instance of Invenio follow the installation [here](https://invenio.readthedocs.io/en/latest/quickstart/quickstart.html)
+
+## Community
+(insert contact of a possible comunity, like gitter or slack)
+
+## Code of conduct
+[Code of conduct](https://www.contributor-covenant.org/)
+
+## Reporting a bug and asking help
+
+If you found a bug plese report it [here](https://github.com/open-education-polito/fare-invenio) opening an issue.
+Searching help? (contatti per chi chiede aiuto)
+
+## Semantic Versioning
+[Here](https://semver.org/) the semantic versioning we use
+
+# Maintenance
+
+Code actually maintained by (add contacts of maintainers)
+
+# License
+
+This code is licensed under a GNU Affero General Public License, v3. See the LICENSE.md file for extended info, and the license files inside each module's repository.
```

Comment:
I am not sure we are allowed to do so, we need to double check this. @marioscovazzi

Reply:
MIT confirmed.

Reply:
fixed